src/scheduler.cc

```diff
@@ -885,7 +893,7 @@ void Scheduler::terminate_execution(Locker& locker, ExitState exit) {
 void Scheduler::tick(Locker& locker, int64 now) {
   tick_schedule(locker, now, true);
 
-  int first_non_empty_ready_queue = -1;
+  int first_non_empty_ready_queue = NUMBER_OF_READY_QUEUES;
```

Comment:
This makes the code from line 903-908 dead. Remove.

Reply:
yes.
I wanted to remove those lines anyway.
done.